src/components/checkbox/index.js

```diff
@@ -115,8 +115,11 @@ class Checkbox extends BaseComponent {
 
   render() {
     const {value, selectedIcon, style, color, iconColor, disabled, testID, ...others} = this.getThemeProps();
+    const checkboxValueForDetox = value ? 'checked' : 'unchecked';
```

Comment:
I'm not a fan of having `detox` in the variable name.
maybe just call it `accessibilityLabel`.